src/basic/missing_hidraw.h

```diff
@@ -0,0 +1,9 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#include <linux/hidraw.h>
+
+/* Proposed, pending */
```

Comment:
Is this not merged in the kernel yet then?

Reply:
No, it's not - do you want me to switch to draft? or is there a "pending" tag?

Reply:
AFAIK this is merged into 6.12, right? Please drop this comment, and we can merge this.

Reply:
Updated, thanks